components/workflow.rst

```diff
@@ -58,33 +58,12 @@ The ``Workflow`` can now help you to decide what *transitions* (actions) are all
 on a blog post depending on what *place* (state) it is in. This will keep your domain
 logic in one place and not spread all over your application.
 
-When you define multiple workflows you should consider using a ``Registry``,
-which is an object that stores and provides access to different workflows.
-A registry will also help you to decide if a workflow supports the object you
-are trying to use it with::
-
-    use Acme\Entity\BlogPost;
-    use Acme\Entity\Newsletter;
-    use Symfony\Component\Workflow\Registry;
-    use Symfony\Component\Workflow\SupportStrategy\InstanceOfSupportStrategy;
-
-    $blogPostWorkflow = ...;
-    $newsletterWorkflow = ...;
-
-    $registry = new Registry();
-    $registry->addWorkflow($blogPostWorkflow, new InstanceOfSupportStrategy(BlogPost::class));
-    $registry->addWorkflow($newsletterWorkflow, new InstanceOfSupportStrategy(Newsletter::class));
-
 Usage
 -----
 
-When you have configured a ``Registry`` with your workflows,
-you can retrieve a workflow from it and use it as follows::
-
     // ...
     // Consider that $blogPost is in place "draft" by default
     $blogPost = new BlogPost();
-    $workflow = $registry->get($blogPost);
```

Comment:
$workflow is not defined anymore

Reply:
it is defined in line 55.

Reply:
Yes, but shouldn't we show an example of injecting the workflow into a controller action or a service? There is no example for this, right?

Reply:
Oh I think it's mentioned here:
```
Read more about the usage of the :doc:`Workflow component </workflow>` inside a Symfony application.
```